src/topology/continuous_function/algebra.lean

```diff
@@ -639,6 +652,24 @@ instance has_scalar' {α : Type*} [topological_space α]
   has_scalar C(α, R) C(α, M) :=
 ⟨λ f g, ⟨λ x, (f x) • (g x), (continuous.smul f.2 g.2)⟩⟩
 
+instance {α : Type*} [topological_space α] [locally_compact_space α]
+  {R : Type*} [topological_space R] {M : Type*} [topological_space M]
+  [has_scalar R M] [has_continuous_smul R M] :
+  has_continuous_smul R C(α, M) :=
+⟨begin
+  refine continuous_of_continuous_uncurry _ _,
+  have h : continuous (λ x : (R × C(α, M)) × α, x.fst.snd x.snd) :=
+    continuous_ev.comp ((continuous_snd.comp continuous_fst).prod_mk continuous_snd),
+  exact (continuous_fst.comp ((continuous_fst.comp continuous_fst).prod_mk h)).smul
+    (continuous_snd.comp ((continuous_fst.comp continuous_fst).prod_mk h)),
+end⟩
+
+instance {α : Type*} [topological_space α] [locally_compact_space α]
+  {R : Type*} [topological_space R] {M : Type*} [topological_space M]
+  [has_scalar R M] [has_continuous_const_smul R M] :
+  has_continuous_const_smul R C(α, M) := ⟨λ γ,
+    continuous_of_continuous_uncurry _ (continuous.const_smul continuous_ev γ)⟩
```

Comment:
```suggestion
  has_continuous_const_smul R C(α, M) :=
⟨λ γ, continuous_of_continuous_uncurry _ (continuous_ev.const_smul γ)⟩
```
(make sure to learn the whitespace convention -- `:=` is followed by a newline -- and also the "projection notation")

Reply:
Thanks, done.